src/modules/navigator/rtl_params.c

```diff
@@ -53,13 +53,13 @@
  * @max 150
  * @decimal 1
  * @increment 0.5
- * @group Return To Land
+ * @group Return mode
```

Comment:
@bkueng Note I changed the group here (and below)- hope this is OK!

Reply:
I'm ok with that. Can you capitalize Mode?